utility/manage.py

```diff
@@ -0,0 +1,239 @@
+import os
+import re
+from lxml import etree
+from shutil import rmtree
+from crm_solver.beamlet import Beamlet
+from utility.writedata import WriteData
+from utility.putdata import PutData
+from utility.getdata import GetData
+from utility.accessdata import AccessData
+from utility.exceptions import RenateAuthorizedUserError
+
+
+class Version(object):
+    def __init__(self, value):
+        if not isinstance(value, str):
+            raise TypeError('The expected variable type for Version control is str.')
+        version = value.split(sep='.')
+        if not version.__len__() == 3:
+            raise ValueError('The expected data value is point delimited with 3 expressions. Ex: 1.1.0 ')
+        for val in version:
+            assert isinstance(int(val), int), 'Delimited values are expected to be convertible to <int>.'
+        self.major_version = int(version[0])
+        self.minor_version = int(version[1])
+        self.bugfix_version = int(version[2])
+        self.version = version
```

Comment:
version list is redundant. to be removed

Reply:
Done. In: a3f1cdb

Reply:
Way prettier!

Reply:
Well. Thank you!